pkg/domain/host.go

```diff
@@ -8,33 +8,52 @@ import (
 	"time"
 )
 
-type HostType string
+type Credential struct {
+	Username string `json:"username"`
+	Password string `json:"password"`
+}
 
-const (
-	Domain HostType = "domain"
-	IP     HostType = "ip"
-)
+type Rapporteur struct {
```

Comment:
Rapporteurs should only have a `Name`, `Email` and `IsPrincipal` attribute, no password. This may have been a typo on your side 🤣

```suggestion
type Rapporteur struct {
    Name string `json:"name"`
    Email string `json:"email"`
    IsPrincipal bool `json"is_principal"`
```

Reply:
Yes, it was my mistake, I corrected it.